R/calc_shape_dist.R

```diff
@@ -58,7 +67,12 @@ calc_shape_dist <- function(beta1, beta2,
                             alignment = TRUE,
                             rotation = TRUE,
                             scale = TRUE,
-                            include.length = FALSE) {
+                            include.length = FALSE,
+                            lambda = 0.0) {
+  if (mode == "C" && !scale)
+    cli::cli_abort("Closed curves are currently handled only on the L2
+                   hypersphere. Please set `scale = TRUE`.")
```

Comment:
I would not say L2 hypsersphere as this will be confusing to some users, please change to Hilbert Sphere